gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -684,6 +702,13 @@ public Builder setAllowNonDefaultServiceAccount(boolean allowNonDefaultServiceAc
       return this;
     }
 
+    /** Use DirectPath xDS. Only valid if DirectPath is attempted. */
```

Comment:
Document that an environment variable could override this option.

Reply:
The new API is used to enable DirectPath xDS, but not disable. So I removed the bool parameter. In this way, the environment can not override the option.